utoipa-gen-4.3.0 leaks crate-private type

My `Cargo.toml` is referencing utoipa as follows:

```
[dependencies]
utoipa = { version = "4", features = ["yaml"] }
```

However, compilation of my library fails since version 4.3.0  (dependency on utoipa-gen 4.3.0) with the following compile error:

```
error[E0446]: crate-private type `parse_utils::Str` in public interface
    --> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/utoipa-gen-4.3.0/src/lib.rs:2847:5
     |
2847 |     pub fn parse_next_literal_str_or_include_str(input: ParseStream) -> syn::Result<Str> {
     |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't leak crate-private type
...
2921 |     pub(super) enum Str {
     |     ------------------- `parse_utils::Str` declared as crate-private

For more information about this error, try `rustc --explain E0446`.
```

`utopia-gen` 4.2.0 is working for me.

--- a/utoipa/Cargo.toml
+++ b/utoipa/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "utoipa"
 description = "Compile time generated OpenAPI documentation for Rust"
-version = "4.2.1"
+version = "4.2.2"
 edition = "2021"
 license = "MIT OR Apache-2.0"
 readme = "README.md"
@@ -49,7 +49,7 @@ auto_into_responses = ["utoipa-gen/auto_into_responses"]
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
 serde_yaml = { version = "0.9", optional = true }
-utoipa-gen = { version = "4.2.0", path = "../utoipa-gen" }
+utoipa-gen = { version = "4.3.0", path = "../utoipa-gen" }
 indexmap = { version = "2", features = ["serde"] }
 
 [dev-dependencies]

